redsocks compile error

mv -f gen/version.c.tmp gen/version.c
gcc -MM parser.c main.c redsocks.c log.c http-connect.c socks4.c socks5.c http-relay.c base.c base64.c md5.c http-auth.c utils.c redudp.c dnstc.c gen/version.c 2>/dev/null >.depend || \
	( \
		for I in base.h base64.h dnstc.h http-auth.h libc-compat.h libevent-compat.h list.h log.h main.h md5.h parser.h redsocks.h redudp.h socks5.h utils.h version.h; do \
			export ${I//[-.]/_}_DEPS="`sed '/^\#[ \t]*include \?"\(.*\)".*/!d;s//\1/' $I`"; \
		done; \
		echo -n >.depend; \
		for SRC in parser.c main.c redsocks.c log.c http-connect.c socks4.c socks5.c http-relay.c base.c base64.c md5.c http-auth.c utils.c redudp.c dnstc.c gen/version.c; do \
			echo -n "${SRC%.c}.o: " >>.depend; \
			export SRC_DEPS="`sed '/\#[ \t]*include \?"\(.*\)".*/!d;s//\1/' $SRC | sort`"; \
			while true; do \
				export SRC_DEPS_OLD="$SRC_DEPS"; \
				export SRC_DEEP_DEPS=""; \
				for HDR in $SRC_DEPS; do \
					eval export SRC_DEEP_DEPS="\"$SRC_DEEP_DEPS \$${HDR//[-.]/_}_DEPS\""; \
				done; \
				export SRC_DEPS="`echo $SRC_DEPS $SRC_DEEP_DEPS | sed 's/  */\n/g' | sort -u`"; \
				test "$SRC_DEPS" = "$SRC_DEPS_OLD" && break; \
			done; \
			echo $SRC $SRC_DEPS >>.depend; \
		done; \
	)
make[4]: Leaving directory `/home/main/n16/build_dir/target-mipsel_74kc+dsp2_uClibc-0.9.33.2/redsocks-0.4'
make[4]: Entering directory `/home/main/n16/build_dir/target-mipsel_74kc+dsp2_uClibc-0.9.33.2/redsocks-0.4'
mipsel-openwrt-linux-uclibc-gcc -Os -pipe -mno-branch-likely -mips32r2 -mtune=74kc -mdspr2 -fno-caller-saves -fhonour-copts -Wno-error=unused-but-set-variable -msoft-float -mips16 -minterlink-mips16  -I/home/main/n16/staging_dir/target-mipsel_74kc+dsp2_uClibc-0.9.33.2/usr/include -I/home/main/n16/staging_dir/target-mipsel_74kc+dsp2_uClibc-0.9.33.2/include -I/home/main/n16/staging_dir/toolchain-mipsel_74kc+dsp2_gcc-4.8-linaro_uClibc-0.9.33.2/usr/include -I/home/main/n16/staging_dir/toolchain-mipsel_74kc+dsp2_gcc-4.8-linaro_uClibc-0.9.33.2/include  -g -O2 -std=gnu99 -Wall   -c -o parser.o parser.c
In file included from parser.c:29:0:
utils.h:43:41: warning: 'struct evbuffer' declared inside parameter list [enabled by default]
 char *redsocks_evbuffer_readline(struct evbuffer *buf);
                                         ^
utils.h:43:41: warning: its scope is only this definition or declaration, which is probably not what you want [enabled by default]
utils.h:44:65: error: unknown type name 'evbuffercb'
 struct bufferevent* red_connect_relay(struct sockaddr_in *addr, evbuffercb writecb, everrorcb errorcb, void *cbarg);
                                                                 ^
utils.h:44:85: error: unknown type name 'everrorcb'
 struct bufferevent* red_connect_relay(struct sockaddr_in *addr, evbuffercb writecb, everrorcb errorcb, void *cbarg);
                                                                                     ^
make[4]: *** [parser.o] Error 1
make[4]: Leaving directory `/home/main/n16/build_dir/target-mipsel_74kc+dsp2_uClibc-0.9.33.2/redsocks-0.4'
make[3]: *** [/home/main/n16/build_dir/target-mipsel_74kc+dsp2_uClibc-0.9.33.2/redsocks-0.4/.built] Error 2
make[3]: Leaving directory `/home/main/n16/feeds/packages/net/redsocks'
make[2]: *** [package/feeds/packages/redsocks/compile] Error 2
make[2]: Leaving directory `/home/main/n16'
make[1]: *** [/home/main/n16/staging_dir/target-mipsel_74kc+dsp2_uClibc-0.9.33.2/stamp/.package_compile] Error 2
make[1]: Leaving directory `/home/main/n16'
make: *** [world] Error 2
main@bogon:~/n16$





same problem also in buildbot: 

http://buildbot.openwrt.org:8010/broken_packages/ar71xx/redsocks/compile.txt

--- a/libs/libev/Makefile
+++ b/libs/libev/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libev
 PKG_VERSION:=4.20
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://dist.schmorp.de/libev/Attic/
@@ -44,7 +44,9 @@ CONFIGURE_ARGS += \
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/event.h $(1)/usr/include/ev_event_compat.h
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/ev.h $(1)/usr/include/
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/ev++.h $(1)/usr/include/
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libev.{a,so*} $(1)/usr/lib/
 endef

